src/views/NotFound.vue

```diff
@@ -59,11 +59,18 @@ export default defineComponent({
 .not-found {
   --timingFunction: cubic-bezier(.785, .135, .15, .86);
   height: calc(100vh - var(--headerHeight));
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
 
   h1,
   .circle {
     opacity: 0;
     animation: .75s var(--timingFunction) forwards fadeIn;
+    display: flex;
+    align-items: center;
+    justify-content: center;
```

Comment:
same comment, i don't think we want to target `h1` with the flex styles.

Reply:
Ahh thanks